resources/letters/models/letter_editable.yml

```diff
@@ -1,5 +1,4 @@
 allOf:
-  - $ref: "letter_base.yml"
```

Comment:
Wait, why are we getting rid of this ref to `letter_base` if we're adding `return_envelope` to `letter_base`? Is `return_envelope` only present in letter responses, rather than editable letters? If so, why aren't we adding it to `letter` instead of `letter_base`, and keeping this reference to the latter here?

Reply:
The reason is that we have 4 versions of Letter. 3 are read versions as described in Letter are certified, registered, and no_extra_service, which all extend letter base. The 4th version, editable, did extend from base, however, the behavior of return envelope differs between read and write operations.

When writing, it has two possible value types:
```yaml
oneOf:
  - type: string
  - type: boolean
```
whereas when reading, it has a different set of two possible value types:
```yaml
oneOf:
  - type: boolean
  - type: object
```